.circleci/config.yml

```
@@ -113,7 +113,8 @@ jobs:
     <<: *unix_box
     steps:
       - checkout
-      - <<: *restore_dependency_cache_unix
+      # Don't restore npm cache since we always need latest #master branch
+      - run: npm install
```

What about doing it this way instead?

```suggestion
      - <<: *restore_dependency_cache_unix
      # Always run on the latest master branch, regardless of cache
      - run: npm install act-rules/act-rules.github.io#master
```